### PR TITLE
Add option to order vulnerabilities by id, title or risk_rating

### PR DESCRIPTION
**Pull request by elyousfi5**
### **PR Description**

**Purpose**: This pull request adds functionality to allow users to order vulnerabilities by `id`, `title`, or `risk_rating` when listing vulnerabilities for a specific scan. This enhancement improves the usability of the vulnerability listing feature by providing more flexibility in how the results are presented.

**Changes Made**:

* Introduced a new `order_by` parameter in the `list_vulnz` method, with a default value of `risk_rating`.  
* Implemented sorting logic to handle three ordering options: `id`, `title`, and `risk_rating`.  
* Updated the query to dynamically sort vulnerabilities based on the specified `order_by` parameter.  
* Ensured that if an invalid `order_by` value is provided, the method defaults to sorting by `risk_rating`.

**Benefits**:

* Enhances the user experience by allowing more control over the presentation of vulnerability data.  
* Enables users to easily find vulnerabilities based on their preferred sorting criteria, facilitating quicker analysis and decision-making.

### CODE REVIEW

**Review by benyissa — APPROVED** [review]
LGTM. Good job!

File: src/ostorlab/runtimes/local/runtime.py [hunk 1]
```diff
@@ -658,7 +659,22 @@ def list_vulnz(
...
+                        ],
+                        else_=len(risk_rating.RATINGS_ORDER),
+                    )
+                    vulnerabilities = query.order_by(
+                        case_ordering, models.Vulnerability.title
+                    ).all()
+                elif order_by == "title":
+                    vulnerabilities = query.order_by(models.Vulnerability.title).all()
+                elif order_by == "id":
+                    vulnerabilities = query.order_by(models.Vulnerability.id).all()
```
Comment (3asm) [inline src/ostorlab/runtimes/local/runtime.py:676]: Else a warning would be nice.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/runtimes/local/runtime.py (hunk 1 above)
Comment (elyousfi5) [reply src/ostorlab/runtimes/local/runtime.py:676]: it will never be one other than those 3 !! this is handled by click.Choice already

**Review by adnaneserrar — APPROVED** [review]

File: src/ostorlab/runtimes/local/runtime.py [hunk 2]
```diff
@@ -635,8 +635,9 @@ def install(self, docker_client: Optional[docker.DockerClient] = None) -> None:
     def list_vulnz(
         self,
         scan_id: int,
-        filter_risk_rating: Optional[List[str]],
-        search: Optional[str],
+        filter_risk_rating: Optional[List[str]] = None,
+        search: Optional[str] = None,
+        order_by: str = "risk_rating",
```
Comment (mohsinenar) [inline src/ostorlab/runtimes/local/runtime.py:640]: const!

**Review by 3asm — APPROVED** [review]

**Pull request merged**